x/wasm/types/types.pb.go

```diff
@@ -140,35 +141,35 @@ type AccessConfig struct {
 	Address    string     `protobuf:"bytes,2,opt,name=address,proto3" json:"address,omitempty" yaml:"address"`
 }
 
-func (m *AccessConfig) Reset()         { *m = AccessConfig{} }
-func (m *AccessConfig) String() string { return proto.CompactTextString(m) }
+func (a *AccessConfig) Reset()         { *a = AccessConfig{} }
```

Comment:
Please do not modify generated `*pb.go` files beside automated formatting or goimports. This will be overwritten by the next generator run and causes a lot confusion in code reviews.
Please revert the changes